src/test/java/uk/gov/pay/api/service/GetPaymentServiceTest.java

```diff
@@ -133,10 +123,15 @@ public void testGetPaymentWithCorporateCardSurcharge() {
 
     @Test
     @PactVerification({"connector"})
-    @Pacts(pacts = {"publicapi-connector-get-payment-with-awaiting-capture-request-state"})
+    @Pacts(pacts = {"publicapi-connector-get-payment-with-awaiting-capture-request-state-and-gateway-transaction-id"})
```

Comment:
I think this could be "publicapi-connector-get-payment" as the eventuality is to consolidate all the existing pacts into a single "Get payment" scenario.

Reply:
Rename the method (`testGetPaymentWithChargeInAwaitingCaptureRequest`) below too

Reply:
Done